plugins/system/guidedtours/src/Extension/GuidedTours.php

```diff
@@ -132,25 +104,29 @@ public function startTour(Event $event)
     public function onBeforeCompileHead()
     {
         $app  = $this->getApplication();
-        $doc  = $app->getDocument();
-        $user = $app->getIdentity();
 
-        if ($user != null && $user->id > 0) {
-            Text::script('JCANCEL');
-            Text::script('PLG_SYSTEM_GUIDEDTOURS_BACK');
-            Text::script('PLG_SYSTEM_GUIDEDTOURS_COMPLETE');
-            Text::script('PLG_SYSTEM_GUIDEDTOURS_COULD_NOT_LOAD_THE_TOUR');
-            Text::script('PLG_SYSTEM_GUIDEDTOURS_NEXT');
-            Text::script('PLG_SYSTEM_GUIDEDTOURS_START');
-            Text::script('PLG_SYSTEM_GUIDEDTOURS_STEP_NUMBER_OF');
-            Text::script('PLG_SYSTEM_GUIDEDTOURS_TOUR_ERROR');
-
-            $doc->addScriptOptions('com_guidedtours.token', Session::getFormToken());
-
-            // Load required assets
-            $doc->getWebAssetManager()
-                ->usePreset('plg_system_guidedtours.guidedtours');
+        if (!$app->isClient('administrator') || !$app->getIdentity()->id)
+        {
```

Comment:
```suggestion
        if (!$app->isClient('administrator') || !$app->getIdentity()->id) {
```
Fix PHPCS failing in drone.